homeassistant/const.py

```diff
@@ -163,6 +163,7 @@
 SERVICE_TURN_ON = 'turn_on'
 SERVICE_TURN_OFF = 'turn_off'
 SERVICE_TOGGLE = 'toggle'
+SERVICE_SET_BRIGHTNESS = 'set_brightness'
```

Comment:
Please just define this constant in `light/__init__.py`. No need to generalize it.